crates/interpreter/src/valid.rs

```diff
@@ -764,17 +810,60 @@ impl<'a, 'm> Expr<'a, 'm> {
         Ok(())
     }
 
-    fn br_label(&self, l: LabelIdx) -> Result<ResultType<'m>, Error> {
+    fn stitch(&mut self, branch: &SideTableBranch) {
+        let current_parser = self.parser.save().as_ptr() as isize;
+        let SideTableBranch { parser, side_table_len } = branch;
+        let entry = &mut self.side_table[*side_table_len];
+        if entry.is_none() {
+            *entry = Some(SideTableEntryView {
+                delta_ip: (current_parser - parser.as_ptr() as isize) as i32,
```

Comment:
`(current.parser.as_ptr() as isize - branch.parser.as_ptr() as isize) as i32`
We could have a `SideTableBranch::delta_ip()` helper for that.